trunk/SUAVE/Analyses/Stability/Stability.py

```diff
@@ -83,8 +83,8 @@ def finalize(self):
 
         Properties Used:
         N/A
-        """          
-
+        """

```

Comment:
unnecessary change

Reply:
I reviewed this file. I don't think any code was changed, just the spaces

Reply:
I mean:
git checkout develop trunk/SUAVE/Analyses/Stability/Stability.py